src/adapters/TelnetAdapter.ts

```diff
@@ -576,6 +587,17 @@ export class TelnetAdapter {
                     children = this.getObjectChildren(expression, value.trim());
                 }
 
+                if (['rostring', 'roint', 'rointeger', 'rofloat', 'rodouble', 'roboolean'].includes(lowerExpressionType)) {
```

Comment:
Here's the core part of this PR. We need special handling for these data types, otherwise they get treated as objects.

Reply:
Looks like we are missing rolonginteger and rointrinsicdouble? Or is this handled in some other way?

Reply:
good catch. I added those.